VIDEC Onboarding

vicire1 opened the pull request:
Creating estate app

vicire1 committed:
[ADD] Estate: create the app

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1001.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1001)

vicire1 committed:
[IMP] Estate: Creating table fields

vicire1 committed:
[IMP] Estate: Grant access group user

[FIX] Add licence and correct style

Mathilde411 commented on estate/security/ir.model.access.csv:
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_estate_group_user,estate.group.user,model_estate_property,base.group_user,1,1,1,1
```
End of file newline missing

Mathilde411 commented on estate/views/estate_menus.xml:
```diff
@@ -0,0 +1,8 @@
+            <menuitem id="estate_model_menu_action" action="estate_property_model_action"/>
+        </menuitem>
+    </menuitem>
+</odoo>
```
End of file newline missing

Mathilde411 commented on estate/views/estate_property_views.xml:
```diff
@@ -0,0 +1,8 @@
+        <field name="res_model">estate.property</field>
+        <field name="view_mode">list,form</field>
+    </record>
+</odoo>
```
End of file newline missing

vicire1 committed:
[IMP] Estate: Create views

vicire1 committed:
[IMP] Estate: Create tag, offer, type with according views

[FIX] Changing inverse one2many to property_id

[CORRECTION]

[CORRECTION COMMIT]

add line

vicire1 committed:
[IMP] Estate: Adding computed fields

delete substract

vicire1 committed:
[IMP] Estate: Creating buttons

[FIX] delete whitespaces

vicire1 committed:
[IMP] Estate: Adding constraints on fields

[FIX] Mofify usererror exception

vicire1 committed:
[IMP] Estate: Add the sprinkles

vicire1 committed:
[IMP] Estate: Adding inheritance

[FIX] super()

vicire1 committed:
[IMP] Estate: Create invoice when sold

[FIX] style

vicire1 committed:
[IMP] Estate: Create kanban view for estate proprety model

[FIX] Refactor

[FIX] Correct style
